apps/server/src/modules/h5p-editor/service/h5p-translator.service.ts

```diff
@@ -0,0 +1,35 @@
+import i18next from 'i18next';
+import path from 'path';
+import { ITranslationFunction } from '@lumieducation/h5p-server/build/src/types';
+import i18nextFsBackend from 'i18next-fs-backend';
+
+export const Translator = {
+	async translate() {
+		const pathBackend = path.join(
+			__dirname,
+			'../../../../../../node_modules/@lumieducation/h5p-server/build/assets/translations/{{ns}}/{{lng}}.json'
+		);
```

Comment:
Probably better than relying on relative paths

```suggestion
		const lumiPackagePath = path.dirname(require.resolve('@lumieducation/h5p-server/package.json'));

		const pathBackend = path.join(lumiPackagePath, 'build/assets/translations/{{ns}}/{{lng}}.json');
```